test/bdd/bddtests_test.go

```diff
@@ -60,13 +60,15 @@ func TestMain(m *testing.M) {
 					panic(fmt.Sprintf("Error composing system in BDD context: %s", err))
 				}
 
-				testSleep := 80
+				testSleep := 100
```

Comment:
Is this necessary? I recently pushed a change that reduces the retry interval for AMQP so we shouldn't have to wait that long.

Reply:
I was noticing that sometimes the command to generate the document for IPFS would fail (it calls an endpoint on orb-domain1 and would sometimes return a connection refused error). Let me take another look at this. Maybe I should add retries instead?

Reply:
I've reverted the sleep time and added retries instead around the code that was occasionally failing. So far it seems to be working (however, I haven't see the CLI had to use the retries yet, so I suppose I'll just have to keep an eye on this)